### PR TITLE
70821069 - feat: init checkbox component in to shared/ui

### PR DESCRIPTION
**Pull request by EvaM25**
Task: https://github.com/users/MariiaBel/projects/3/views/1?pane=issue&itemId=70821069

Can't connect checkbox.css in storybook.  Check it out, please!

### CODE REVIEW

File: src/06_shared/img/svg/check.svg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <svg viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg" fill="none">
-<path d="M19 5L8.6875 16L4 11" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M19 5L8.6875 16L4 11" stroke="url(#paint0_linear_176_9088)" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (MariiaBel) [inline src/06_shared/img/svg/check.svg:2]: Need to return black svg. Use mask for making gradient https://www.w3schools.com/css/css3_masking.asp

File: src/06_shared/ui/checkbox/checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.checkbox {
```
Comment (MariiaBel) [inline src/06_shared/ui/checkbox/checkbox.scss:1]: We are using BEM. Do not style tags

File: src/06_shared/ui/checkbox/checkbox.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  input{
+    opacity: 0;
+    width: 0;
+    height: 0;
+
+  &:checked + span::after{
+    display: block;
+  } 
+  &:focus + span::before{
+  border-color: #9747ff;
```
Comment (MariiaBel) [inline src/06_shared/ui/checkbox/checkbox.scss:44]: tab

File: src/stories/Checkbox.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import "./checkbox.css"
```
Comment (MariiaBel) [inline src/stories/Checkbox.js:1]: change link  on your component scss

File: src/stories/checkbox.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
+.container{
```
Comment (MariiaBel) [inline src/stories/checkbox.css:1]: Need to remove that file/
DRY/ don't repeat yourself

File: src/stories/Checkbox.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+export const createCheckbox = ({ checked = false}) => {
+    const check = checked ? "checked" : "";
+
+
+return  `<div class="ckeckbox"><input type="checkbox" class="checkbox" ${check}><label class="checkbox"assets
+<svg class="checkbox" width="25" height="25" viewBox="0 0 25 25" xmlns="http://www.w3.org/2000/svg" fill="none">
```
Comment (MariiaBel) [inline src/stories/Checkbox.js:8]: 2 classes with same name class="checkbox"

File: src/06_shared/ui/checkbox/checkbox.scss (hunk 2 above)
Comment (EvaM25) [reply src/06_shared/ui/checkbox/checkbox.scss:1]: It's not a tag. It's a class.

Comment (MariiaBel) [reply src/06_shared/ui/checkbox/checkbox.scss:1]: You style tags inside the class .checkbox

**Comment by MariiaBel** [comment]
Need to fix that bug
image

**Review by MariiaBel — APPROVED** [review]

**Pull request merged**